chart/elastalert2/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: Automated rule-based alerting for Elasticsearch
 name: elastalert2
-version: 2.4.0
-appVersion: 2.4.0
+version: 2.4.1
```

Comment:
Please revert this change. The project maintainer will assign new version numbers at the time of the next release.